PurchasesTests/Attribution/AttributionPosterTests.swift

```diff
@@ -198,115 +194,101 @@ class AttributionPosterTests: XCTestCase {
         expect(MockAdClientProxy.requestAttributionDetailsCallCount) == 0
     }
 
-    func testPostAppleSearchAdsAttributionIfNeededPostsIfAuthorizedOnNewOS() {
-        #if os(iOS)
-        if #available(iOS 14, *) {
-            systemInfo.stubbedIsOperatingSystemAtLeastVersion = true
-            
-            MockTrackingManagerProxy.mockAuthorizationStatus = .authorized
-            MockAttributionTypeFactory.shouldReturnAdClientProxy = true
-            MockAttributionTypeFactory.shouldReturnTrackingManagerProxy = true
+    func testPostAppleSearchAdsAttributionIfNeededPostsIfAuthorizedOnNewOS() throws {
+        guard #available(iOS 14, *) else { throw XCTSkip() }
```

Comment:
I think the os(iOS) check is still needed, otherwise this will still run, because ` #available(iOS 14, *)` will (incredibly) return true on tvOS 14+. 
Don't know a better way to handle this other than the OS check. 

You can double-check this by selecting the RevenueCat scheme and a tvOS simulator, then running tests

Reply:
I figured that was the reason, I wanted to see if CI caught it 😛

Reply:
Fixed.